lcd/emonPiLCD.py

```diff
@@ -217,14 +217,6 @@ class Background(threading.Thread):
     def __init__(self):
         threading.Thread.__init__(self)
         self.stop = False
-	self.set_defaults()
```

Comment:
I think you need to do a pull before you push. It looks like you're deleting some of the code I added.